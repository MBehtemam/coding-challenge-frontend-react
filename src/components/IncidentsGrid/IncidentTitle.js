import styled from 'styled-components';

const Title = styled.h1`
  color: rgb(48, 48, 48);
  font-size: 2rem;
  padding-bottom: 10px;
`;

export default Title;
