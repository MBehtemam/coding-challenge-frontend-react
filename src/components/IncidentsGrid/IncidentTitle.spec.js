import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import 'jest-styled-components';
import IncidentTitle from './IncidentTitle';

describe('Test suits for Incident Title', () => {
  it('should match with snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <IncidentTitle />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
