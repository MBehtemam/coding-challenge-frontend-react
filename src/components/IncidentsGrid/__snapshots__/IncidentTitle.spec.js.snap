// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Incident Title should match with snapshot 1`] = `
.c0 {
  color: rgb(48,48,48);
  font-size: 2rem;
  padding-bottom: 10px;
}

<h1
  className="c0"
/>
`;
